app/src/main/java/org/gdg/frisbee/android/fragment/SwipeRefreshRecyclerViewFragment.java

```diff
@@ -165,8 +165,8 @@ private boolean canListViewScrollUp(RecyclerView recyclerView) {
                     return position != 0;
                 } else if (layoutManager instanceof StaggeredGridLayoutManager) {
                     int[] positions = ((StaggeredGridLayoutManager) layoutManager).findFirstCompletelyVisibleItemPositions(null);
-                    for (int i = 0; i < positions.length; i++) {
-                        if (positions[i] == 0) {
+                    for (int position : positions) {
```

Comment:
Android Studio suggested it. I think in fact it improves it a little bit because in the previos one `position.length` is called every time.